README.md

```diff
@@ -104,6 +104,27 @@ ObservableTransformer<RelativeDto, Relative> toUILayerModel() { /* ... */ }
 
 ObservableTransformer<Relative, List<Siblings>> groupSiblings() { /* ... */ }
 ```
+
+## Komprehensions-reactor
+
+Komprehensions-reactor is an extension module that allows chaining of [Project Reactor](https://projectreactor.io/) `Flux` and `Mono` operators.
+
+### Map comprehensions
+
+Komprehensions-reactor contains functions `doFlatMap()` for `flatMap()`, `doConcatMap()` for `concatMap()`, `doSwitchMap()` for `switchMap()`. Each takes from 1 to 9 function each with an increasing number of parameters, and returns an `Flux` of the type of the return of the last function. It also contains functions `doFlatMapMono()` for `flatMap()` on the `Mono` operator. Each takes from 1 to 8 function each with an increasing number of parameters, and returns a `Mono` of the type of the return of the last function.
+
+```java
+Flux<String> getUserFriends =
+    // chained with flatMap()
+    RxComprehensions.doFlatMap(
```

Comment:
ReactorComprehensions?

Reply:
🤦‍♂️ Good catch lol, on it

Reply:
Also, are the whole docs wrong? hahaha

Reply:
Well actually in looking at it, shouldn't it be `KomprehensionsReactor` as that is the `JVMName I gave that file? And the Rx `README` entries should be `KomprehensionsRx` to match their JVM names? This is a question, because I'm not positive if that is correct.

Reply:
Yep, could you please do a quick fixeroo on the readme?

Reply:
👍 updated